platform/darwin/src/NSComparisonPredicate+MGLAdditions.h

```diff
@@ -0,0 +1,9 @@
+#import <Foundation/Foundation.h>
+
+#include <mbgl/style/filter.hpp>
+
+@interface NSComparisonPredicate (MGLAdditions)
+
+- (mbgl::style::Filter)mgl_filter;
```

Comment:
Since this is already declared on NSPredicate(MGLAdditions), there's no need to declare it here and in NSCompoundPredicate(MGLAdditions).